### ISSUE TITLE
Project Disappears and Can't stop server

### ISSUE THREAD

**Issue by nickeve** [phonegap/phonegap-app-desktop#437]
I went through the introductory documentation here http://docs.phonegap.com/ and created the hello world app.  I then tried editing index.html and was able to make some minor changes. A few minutes in, the Desktop App crashed and my project no longer shows up in the Desktop.  The project just disappeared.  I cannot re add an existing project  as when I do it says that project already exists, though the desktop app doesn't show it.  It says the server is running, but there is no way to stop the server as myapp isn't shown in the Desktop client (so no start stop buttons) and I cannot figure out how to do it from the CLI tools (as it was started in the Desktop client, not a terminal, so there is no Control C option to stop it. 

What do I do?  Just remove and reinstall everything?  How do I stop the server? 

My configuration: OSX 10.10.2.  Running the latest version of all phonegap elements (all were downloaded and updated today).

**Comment by hermwong** [comment]
When you close the Desktop App it will close the server process.

Is it just displaying an empty Desktop App with the green bar at the bottom saying the server is running?

Would you be able to provide a screenshot?

**Comment by nickeve** [comment]
Ok, strangely closing the app didn't seem to be stopping the server process.  Every time it would open it would be as you said an empty screen with the green bar at the bottom saying the server is running. I even tried rebooting and it was still saying that.  I then deleted and reinstalled everything, and was able to copy the old code in and get it working, so I'm not sure what exactly happened.  Now that it works, I can't provide a screenshot.  Thanks for your help, sorry to take up your time. I should have closed this out as soon as I resolved my issue.
